sktm/jenkins.py

```diff
@@ -213,27 +227,36 @@ def build(self, jobname, baserepo=None, ref=None, baseconfig=None,
         if emails:
             params["emails"] = ",".join(emails)
 
-        if patchwork:
-            params["patchwork"] = " ".join(patchwork)
+        if patch_url_list:
+            params["patchwork"] = " ".join(patch_url_list)
 
         if makeopts is not None:
             params["makeopts"] = makeopts
 
         logging.debug(params)
-        job = self.server.get_job(jobname)
-        expected_id = self.server.get_job(jobname).get_next_build_number()
-        self.server.build_job(jobname, params)
-        build = self.find_build(jobname, params, expected_id)
+        job = self.server.get_job(self.name)
+        expected_id = self.server.get_job(self.name).get_next_build_number()
+        self.server.build_job(self.name, params)
+        build = self.__find_build(params, expected_id)
         logging.info("submitted build: %s", build)
         return build.get_number()
 
-    def is_build_complete(self, jobname, buildid):
-        job = self.server.get_job(jobname)
+    def is_build_complete(self, buildid):
+        """
+        Check if a project build is complete.
+
+        Args:
+            buildid:    Jenkins build ID to get the status of.
+
+        Return:
+            True if the build is complete.
```

Comment:
We should mention `else` case here too, especially since by default `None` is returned, which is not the same thing as `False`.

Reply:
I'm adding `, False if not`, but I don't see how `None` could be returned. I thought `not` can only result in `True` or `False`, or have I misunderstood something?

Reply:
Yes, but it's not clear from the docstring. Based on it, it can be a `if complete return True` and nothing else, and in case the condition wouldn't pass `None` would be returned as a default. So it's an inconsistency between the docstring and code. Adding `, False if not` describes what happens much better :)